File: WebAppCore/Integration/Pharmacy/Service/ObjectionsService.cs
using Integration.Pharmacy.Model;
using Integration.Pharmacy.Repository;
using System;
using System.Collections.Generic;
using System.Text;
using RestSharp;

namespace Integration.Pharmacy.Service
{
   public class ObjectionsService
    {
        ObjectionsRepository objectionsRepository = new ObjectionsRepository();

        public List<Objection> GetAll()
        {
            return objectionsRepository.GetAll();
        }

        public void saveEntity(Objection newObjection)
        {
            objectionsRepository.saveEntity(newObjection);
        }
    }
}

File: PharmacyApp/src/main/java/com/pharmacy/service/ObjectionsService.java
package com.pharmacy.service;

import java.util.List;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.http.HttpEntity;
import org.springframework.http.HttpHeaders;
import org.springframework.http.HttpMethod;
import org.springframework.http.ResponseEntity;
import org.springframework.stereotype.Service;
import org.springframework.web.client.RestTemplate;

import com.pharmacy.dto.ObjectionDto;
import com.pharmacy.model.Medicine;
import com.pharmacy.model.Objection;
import com.pharmacy.repository.MedicineRepository;
import com.pharmacy.repository.ObjectionRepository;
import com.pharmacy.repository.ObjectionRepositoryImpl;

@Service
public class ObjectionsService {

	private static final String API_URL = "http://localhost:43818/";
    private final RestTemplate restTemplate;
	
    @Autowired
    public ObjectionsService() {
        this.restTemplate = new RestTemplate();
    }
	
	private ObjectionRepositoryImpl objectionRepository = new ObjectionRepositoryImpl();


	public void saveObjection(Objection newObjection) {
		
		objectionRepository.save(newObjection);		
	}
    
}
